test_programs/cl5/1.lox

```diff
@@ -0,0 +1,27 @@
+---
+expected_error_type: none
+---
+class Animal {
+  makeSound() {
+    print this.sound;
+  }
+  
+  identify() {
+    print this.species;
+  }
+}
+
+var dog = Animal();
+dog.sound = "Woof";
+dog.species = "Dog";
+
+var cat = Animal();
+cat.sound = "Meow";
+cat.species = "Cat";
+
+// Swap methods between instances
+cat.makeSound = dog.makeSound;
+dog.identify = cat.identify;
```

Comment:
TODO: Think about whether this needs to be a differnt stage, or merged into previous. 

New things that we're testing here: 

- Accessing a method as a value (a.b, without a.b())
- Whether methods are bound to their instance

Reply:
We should merge this into the next stage (the "The this keyword" stage). It can be test-2 or test-3.